### ISSUE TITLE
Create Order Flow of payment using Node

### ISSUE THREAD

**Issue by himanshusharma89** [himanshusharma89/relic_bazaar#40]
- [ ] Create the endpoint for the orders webhook URL using Node
- [ ] Host the functions
- [ ] Call these methods in the app

We can use Cloud Functions for that, but it requires a Blaze plan on Firebase Console, need to find an alternate way of doing so.

**Comment by pranavi79** [comment]
I am assuming no one is assigned to this issue, can I work on it?

**Comment by himanshusharma89** [comment]
> I am assuming no one is assigned to this issue, can I work on it?

Yeah, no one is assigned. I guess I have mentioned the reason why in the Discord Project Channel

**Comment by himanshusharma89** [comment]
@pranavi79 Do you want to work on this issue?

**Comment by alam-shoaib** [comment]
is it still open?

**Comment by himanshusharma89** [comment]
Yes @alam-shoaib. Should I assign it to you?

**Comment by alam-shoaib** [comment]
> Yes @alam-shoaib. Should I assign it to you?

yes please

**Comment by alam-shoaib** [comment]
> Yes @alam-shoaib. Should I assign it to you?

Can u please give a brief discription on what it is to be done and how to go about it

**Comment by alam-shoaib** [comment]
@himanshusharma89 please give a brief description of about what i have to do?
which file to work on where are backend files located

**Comment by himanshusharma89** [comment]
@alam-shoaib Basically, we need to deploy certain methods which will handle the payments flow with any payment gateway provider either stripe or razorpay.

These functions generally handle encryption before sending the request and vice versa.

However, since the time of this issue being created you can explore firebase extensions or any other alternative

**Comment by alam-shoaib** [comment]
can i use Stripe? @himanshusharma89

**Comment by himanshusharma89** [comment]
Yes you can @alam-shoaib
